### PR TITLE
Blendmodes

### PR DESCRIPTION
**Pull request by skoa-h**
I tried to implement it compatible. Had hard times with color burn & dodge and soft light, so they are still missing, but now we have quite some blend modes available.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,5 @@ node_modules
 /test/output/*.bmp
 
 .DS_Store
+.settings
+/update_from_upstream.bat
```
Comment (strandedcity) [inline .gitignore]: this seems like a script that's particular to your computer. Can we remove this line?

Comment (skoa-h) [inline .gitignore]: Of course

**Comment by godong9** [comment]
Hi, I wonder why this PR wasn't merged. 
I need this feature but I can't use this.

**Comment by hipstersmoothie** [comment]
closing in favor of #191

**Pull request closed without merging**